### PR TITLE
coafile.rst: Add aspects section

### PR DESCRIPTION
**Pull request by adhikasp**
Relates to https://github.com/coala/coala/issues/4665

### CODE REVIEW

**Comment by jayvdb** [comment]
CI failing

**Comment by jayvdb** [comment]
Is this example real?
I believe it wont work, as no bear understands this yet.
So, this documentation needs to be merged only if a bear is enhanced.

**Comment by adhikasp** [comment]
I think I will change it to be more abstract (showing dummy config/non real aspect like above in this file). After a bear got aspectized, I will add the Tutorial.rst for the working, real example

File: Users/coafile.rst [hunk 1]
```diff
@@ -107,6 +107,31 @@ Instead, make the inheritance explicit::
...
+to explicitly state list of bears, coala will choose it automatically based on
+requested aspects in coafile. To run coala in this mode, we need to define
+`aspects`, `files`, `languages`, and optionally aspect tastes setting. See
+the following example::
+
+  [all]
+  files = **
+  aspects = aspectname1, AspectName2 # case-insensitive
+  # defining an aspect's taste
+  aspectname1.aspect_taste = 80
```
Comment (jayvdb) [inline Users/coafile.rst]: `.` should be `:`

**Comment by jayvdb** [comment]
ack 682c13d

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**